catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/SeedCommandTest.java

```diff
@@ -285,4 +313,64 @@ public boolean matches(Object o) {
             return test.test((QueryRequest) o);
         }
     }
+
+    private void mockMultipleCatalogFrameworkQueries(int pageSize, int resourceLimit)
+            throws Exception {
+
+        mockResourceResponse();
+        QueryResponse queryResponseMock = mock(QueryResponse.class);
+
+        // Populate list of mock results, sized at resource limit
+        List<Result> resultsList = populateResultMockList(resourceLimit);
+        List<Result>[] queriedResultsList = mockQueryGetResults(resultsList, pageSize);
+
+        when(catalogFramework.query(any())).thenReturn(queryResponseMock);
+        when(queryResponseMock.getResults()).thenReturn(queriedResultsList[0], queriedResultsList);
```

Comment:
The purpose of `mockQueryGetResults` is to make an array that provides `thenReturn` with two queries. The reason being that `queryResponseMock.getResults()` is called twice per iteration, and so it will need to get any given query twice. The desired affect is
```when(queryResponseMock.getResults())
    .thenReturn(queriedResultsList[0])
    .thenReturn(queriedResultsList[0])
    .thenReturn(queriedResultsList[1])
    .thenReturn(queriedResultsList[1]);```